CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## Unreleased
+
+## [4.0] - 2021-05-11
```

Comment:
Could the date format be improved or at least be specified?

Reply:
Actually comparing it with the other changelog entries, it is not correct.

Reply:
@s2oBCN the date needs to be adjusted, it should be 2021-11-05 to be in line with the other entries in the Changelogs.